### ISSUE TITLE
partial_fc_v2.py   how does the backward  work  ,if   margin_softmax  is arcface

### ISSUE THREAD

**Issue by vinjancai** [deepinsight/insightface#2255]
https://github.com/deepinsight/insightface/blob/master/recognition/arcface_torch/partial_fc_v2.py  

165 rows  logits = self.margin_softmax(logits, labels) 

can the margin_softmax(arcface)   backward ?

**Comment by WuKeyu** [comment]
i have a same question. @vinjancai have you found the answer?

**Comment by supertx** [comment]
😢any answer?
